client/views/add/add.js

```diff
@@ -43,6 +43,12 @@ Template.add.events({
   'click .plusbutton': function (event, template) {
     const row = event.currentTarget.parentElement;
     const modBoxes = document.getElementsByClassName('modbox');
+    for (let i = 0; i < modBoxes.length - 1; i++) { // check if the email was already added or not
```

Comment:
I think it makes sense to pull this logic out into a separate function. If it's named carefully, we won't need this comment either.

Reply:
Done